hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/AbstractRpcClient.java

```diff
@@ -527,6 +528,12 @@ public RpcChannel createRpcChannel(ServerName sn, User user, int rpcTimeout)
     return new RpcChannelImplementation(this, createAddr(sn), user, rpcTimeout);
   }
 
+  @Override
+  public RpcChannel createHedgedRpcChannel(Set<ServerName> sns, User user, int rpcTimeout)
+      throws UnknownHostException {
+    throw new NotImplementedException("Should be called on an AbstractRpcClient implementation.");
```

Comment:
I think you mean NettyRpcClient. Simply "method not supported for this implementation." is fine.